src/node/nodestate.h

```diff
@@ -603,6 +603,42 @@ namespace ccf
       }
     }
 
+    void node_quotes(Store::Tx& tx, nlohmann::json& j)
+    {
+      auto nodes_view = tx.get_view(network.nodes);
+
+      nlohmann::json quotes;
+      nodes_view->foreach([&quotes](const NodeId& nid, const NodeInfo& ni) {
+        if (ni.status == ccf::NodeStatus::TRUSTED)
+        {
+          nlohmann::json quote;
+          quote["raw"] = std::string(ni.quote.begin(), ni.quote.end());
+
+#ifdef GET_QUOTE
+          oe_report_t parsed_quote = {0};
+          auto res =
+            oe_parse_report(ni.quote.data(), ni.quote.size(), &parsed_quote);
+          if (res != OE_OK)
+          {
+            LOG_FAIL << "Failed to parse quote: " << oe_result_str(res)
+                     << std::endl;
+          }
+          else
+          {
+            std::stringstream ss;
+            for (auto c : parsed_quote.identity.unique_id)
+              ss << std::hex << std::setw(2) << std::setfill('0')
+                 << static_cast<int>(c);
+            quote["mrenclave"] = ss.str();
```

Comment:
Should we record these magic strings in a separate header file?

Reply:
Yes!

Reply:
Actually we don't seem to do this for any other RPCs, and I think this should be part of the schema work, so I'm going to leave this for a separate PR.

Reply:
Filed as #76